packages/perseus/src/util/colors.ts

```diff
@@ -81,13 +83,9 @@ const KhanColors = {
     GRAY_I: "#21242C",
     KA_BLUE: "#314453",
     KA_GREEN: "#71B307",
-    // Don't actually use _BACKGROUND! Make things transparent instead. The
-    // background color used in exercises is subject to change at the whim
-    // of any redesigns.
-    _BACKGROUND: "#FDFDFD", // TODO(eater): Get rid of this altogether.
-    INTERACTING: "#71B307", // KA_GREEN
-    INTERACTIVE: "#71B307", // KA_GREEN
-    DYNAMIC: "#6495ED", // BLUE
+    INTERACTING: color.green,
+    INTERACTIVE: color.green,
```

Comment:
Could also just combine these into one since they are both the same color. Thoughts?

Reply:
Hmm, there are 45 instances of INTERACTIVE and 26 instances of INTERACTING all across 13 files. I would probably make `INTERACTING` just be `INTERACTIVE`, so it would only add 5 files with changes. 

I don't think it would have wider consequences since both are using the same color, though I don't know if the change in meaning would be troublesome. 

Perhaps this should be part of a wider change of updating `KhanColors` at some point and saved for another time since it has so many instances. I think I assumed it would be a smaller change. 😅

Reply:
Yeah in that case I would hold off and just submit a backlog task to revisit in the future 😉